Sources/Query.swift

```diff
@@ -169,7 +517,175 @@ public struct Query<ContentType: ContentModel> {
     }
 }
 
-extension String {
+/// Queries on Asset types. All methods from Query, and therefore ChainableQuery, are inherited and available.
+public final class AssetQuery: Query {
```

Comment:
Yes, they cannot subclass. It is to prevent a compile error with designated initializers (which i must have with this `protocol` `extension` default implementation architecture. The compile error reads:
```
Initializer requirement 'init()' can only be satisfied by a `required` initalizer in a non-final class 'QueryOn<EntryType>'.
```
Same goes for the other types.